### ISSUE TITLE
survey dropout experimental conditions

### ISSUE THREAD

**Issue by ryuji0123** [ridgei/large_scale_ablation#2]
## Summary
Survey dropout experimental conditions for our reference.

## Goal
When all conditions are listed.

## Todo
- [ ] [Dropout: A Simple Way to Prevent Neural Networks from Overfitting](https://jmlr.org/papers/v15/srivastava14a.html)
- [ ] [Dropout Training, Data-dependent Regularization, and Generalization Bounds](http://proceedings.mlr.press/v80/mou18a.html)
- [ ] [On the Regularization Properties of Structured Dropout](https://openaccess.thecvf.com/content_CVPR_2020/papers/Pal_On_the_Regularization_Properties_of_Structured_Dropout_CVPR_2020_paper.pdf)

## Deadline
07/08

## Parent issue
#1 

## References
None

## Notes
None

**Comment by ryuji0123** [comment]
Dropout: A Simple Way to Prevent Neural Networks from Overfitting

9.2 Logistic Regression and Deep Networks
<img width="958" alt="スクリーンショット 2021-07-08 13 43 51" src="https://user-images.githubusercontent.com/34001189/124863695-89170900-dff2-11eb-92a5-33a1e82f1104.png">

**Comment by ryuji0123** [comment]
In [pytorch](https://github.com/pytorch/pytorch/blob/master/torch/nn/functional.py#L1151), only Bernoulli dropout is implemented.

**Comment by ryuji0123** [comment]
5.2. Classification
Dataset: MNIST

<img width="552" alt="スクリーンショット 2021-07-08 15 04 43" src="https://user-images.githubusercontent.com/34001189/124870532-d77dd500-dffd-11eb-8877-ea36b92d9570.png">

<img width="594" alt="スクリーンショット 2021-07-08 15 03 54" src="https://user-images.githubusercontent.com/34001189/124870455-b87f4300-dffd-11eb-9772-d5e0faa5de47.png">

**Comment by ryuji0123** [comment]
**Self-Normalizing Neural Networks**

<img width="799" alt="スクリーンショット 2021-07-08 15 17 06" src="https://user-images.githubusercontent.com/34001189/124871878-9090df00-dfff-11eb-8bf6-bad5f322c1b0.png">


<img width="805" alt="スクリーンショット 2021-07-08 15 17 20" src="https://user-images.githubusercontent.com/34001189/124871904-98508380-dfff-11eb-81f5-a2b03dd8cff1.png">

**Comment by ryuji0123** [comment]
**Dropout Training, Data-dependent Regularization, and Generalization Bounds**

6. Experiments
<img width="877" alt="スクリーンショット 2021-07-08 15 43 02" src="https://user-images.githubusercontent.com/34001189/124874854-309c3780-e003-11eb-89a3-13eeb549c72b.png">
